src/libclipper/src/redis.cpp

```diff
@@ -438,6 +454,22 @@ bool add_application(redox::Redox& redis, const std::string& appname,
   }
 }
 
+bool add_model_links(redox::Redox& redis, const std::string& appname,
```

Comment:
Actually, I'm going to redistribute the validation responsibilities.
Management frontend:
- Confirm that the user supplied only one model_name
- Make sure that there will only be one link

Redis:
- Confirm that an app with specified app name exists
- Confirm that a model exists for each specified model name
- Confirm input types match

Let me know if that sounds reasonable